stacks/environment-aws/codeservices.tf

```diff
@@ -1,3 +1,27 @@
+data "aws_vpc" "lead_vpc" {
+  tags = {
+    Name = "${var.aws_environment}-lead-vpc"
+  }
+}
+
+data "aws_subnet_ids" "eks_workers" {
+  vpc_id = data.aws_vpc.lead_vpc.id
+
+  filter {
+    name   = "tag:subnet-kind"
+    values = [
+      "private"
+    ]
+  }
+
+  filter {
+    name   = "cidr-block"
+    values = [
+      "*/18"
+    ]
+  }
+}
+
```

Comment:
Can we just add the aws_subnet_ids data to the condservices module instead of passing the variable in?